src/js/providers/tracks-mixin.js

```diff
@@ -401,6 +401,18 @@ define(['../utils/underscore',
     ////// PRIVATE METHODS
     //////////////////////
 
+    function _addCueToTrack(track, vttCue) {
+        if (!utils.isEdge()) {
```

Comment:
But you don't know for sure that it exists in all environments that return true for `isEdge` so it would be safer make sure we're not in Edge or don't have TextTrackCues before trying to use them.

`if (!utils.isEdge() || !window.TextTrackCue) {`